src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -268,6 +315,7 @@ public function itsStateShouldBe(string $state): void
 
     /**
      * @Then it should have shipment in state :state
+     * @Then /^order "[^"]+" should have shipment state "([^"]+)"$/
      */
     public function itShouldHaveShipmentState(string $state): void
```

Comment:
```suggestion
     * @Then /^(it) should have shipment in state "([^"]+)"$/
     * @Then /^(order "[^"]+") should have shipment state "([^"]+)"$/
     */
    public function itShouldHaveShipmentState(OrderInterface $order, string $state): void
```
Should work virtually the same since we're getting an order from shared storage either way